src/services/user.service.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let users = [];
+
+const getAll = () => {
+  return users;
+};
+
+const add = user => {
+  users.push(user);
+};
+
+const getById = (id) => {
+  return users.find(user => user.id === +id) || null;
```

Comment:
```suggestion
  return users.find(user => user.id === Number(id)) || null;
```

The same in other places